### PR TITLE
Update constants

### PR DESCRIPTION
**Pull request by amj**
Changes to constants:
- change max moves to 722

Changes to default flag values
- change resign threshold to -0.999
- change holdout percent from 5% to 3%

Changes to wrapper scripts.
- change resign threshold to -0.999
- change disable-resign amount from 20% to 10%

### CODE REVIEW

File: cluster/evaluator/evaluator_cc_wrapper.sh [hunk 1]
```diff
@@ -30,10 +30,9 @@ echo Retrieiving Models
 gsutil cp ${MODEL_BLACK} .
 gsutil cp ${MODEL_WHITE} .
 
-#TODO use `basename` instead
-BASENAME_BLACK=`echo $MODEL_BLACK  | rev | cut -d/ -f1 | rev`
-BASENAME_WHITE=`echo $MODEL_WHITE  | rev | cut -d/ -f1 | rev`
```
Comment (sethtroisi) [inline cluster/evaluator/evaluator_cc_wrapper.sh]: so much nicer :)

File: cluster/selfplay/cc_wrapper.sh [hunk 2]
```diff
@@ -40,8 +40,8 @@ then
     --model=$NAME \
     --num_readouts=800 \
     --mode=selfplay \
-    --resign_threshold=0.91 \
-    --disable_resign_pct=0.2 \
+    --resign_threshold=0.99 \
+    --disable_resign_pct=0.1 \
```
Comment (brilee) [inline cluster/selfplay/cc_wrapper.sh]: is this a percent or fraction?

File: cluster/selfplay/tpu_wrapper.sh [hunk 3]
```diff
@@ -31,8 +31,8 @@ bazel-bin/cc/main \
   --games_per_inference=16 \
   --parallel_games=32 \
   --num_readouts=800 \
```
Comment (sethtroisi) [inline cluster/selfplay/tpu_wrapper.sh]: did we discuss moving this to 1200?

File: cluster/selfplay/player_wrapper.sh [hunk 4]
```diff
@@ -24,7 +24,7 @@ echo board_size: $BOARD_SIZE
 
 python3 rl_loop.py selfplay \
   --bucket_name=$BUCKET_NAME \
-  --resign-threshold=0.88 \
-  --num_readouts=900
+  --resign-threshold=0.99 \
```
Comment (sethtroisi) [inline cluster/selfplay/player_wrapper.sh]: what are player_wrapper and cc_wrapper used for? and why does their value not agree with the TPU wrapper value (0.999)?

Comment (amj) [reply cluster/selfplay/player_wrapper.sh]: player_wrapper is run by the python selfplay container and cc_wrapper is run by the cc selfplay container.

values harmonized.

File: cluster/selfplay/tpu_wrapper.sh (hunk 3 above)
Comment (amj) [reply cluster/selfplay/tpu_wrapper.sh]: we did, but i'm not sure yet that we'll have the extra quota for v10 (and not v11)

File: cluster/selfplay/cc_wrapper.sh (hunk 2 above)
Comment (amj) [reply cluster/selfplay/cc_wrapper.sh]: it's a fraction o.o  And actually, i didn't check that these flags all still exist this way.

Comment (amj) [reply cluster/selfplay/cc_wrapper.sh]: actually this is true for all the flags. you want s/pct/amt?  something else?

Comment (brilee) [reply cluster/selfplay/cc_wrapper.sh]: s/pct/frac I think

**Pull request merged**